lost/src/main/java/com/mapzen/android/lost/internal/LostClientManager.java

```diff
@@ -212,6 +212,10 @@ private void throwIfClientNotAdded(LostApiClient client) {
     return true;
   }
 
+  @Override public void shutdown() {
```

Comment:
Is `shutdown` an apt naming for this function? How about `clearReportedLocations` or something on those lines?